### DISCUSSION TITLE
Displaying a Live IP Camera video in Avalonia UI

### DISCUSSION THREAD

**Discussion by Junaid-Amjad** [SixLabors/ImageSharp#1541] [Q&A] [answered]
I am looking for the library which help me in displaying the Live IP Camera stream in my Avalonia UI(WPF) Application and later it can save the stream in video format. Can i achieve this using ImageSharp?

**Comment by antonfirsov** [comment, marked as answer]
@Junaid-Amjad based on your description, I doubt you need ImageSharp. We are an image processing library without built-in UI integration.

I think you will get better advice in Avalonia UI forums. Looks like they also have GH discussions set up: https://github.com/AvaloniaUI/Avalonia/discussions
